charts/adot-exporter-for-eks-on-ec2/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: adot-exporter-for-eks-on-ec2
 description: A Helm chart for collecting metrics using ADOT Collector and logs using Fluent Bit to send to AWS monitoring services.
 type: application
-version: 0.8.0
+version: 0.9.0
 appVersion: "0.19.0"
```

```suggestion
appVersion: "0.23.0"
```
Should app version reflect the version of the collector?

---

i agree that